src/components/scores/concept-coach-cell.cjsx

```diff
@@ -10,15 +10,49 @@ ConceptCoachCell = React.createClass
 
   mixins: [CellStatusMixin] # prop validation
 
+  getInitialState: ->
+    placement: 'right'
+
+  setPopoverPosition: ->
```

Comment:
![scores-info-tooltip-flip](https://cloud.githubusercontent.com/assets/954569/13182189/918ee588-d6fe-11e5-91d9-d0242d28e414.gif)

The initial solution was for the popover to always be on the left so it would not go off the table when hovering over item near the right edge of the table.  Then spoke with Robert, he wanted the flyover to not cover the other information in that row/column, which led to current solution.